### DISCUSSION TITLE
are the courses up-to-date?

### DISCUSSION THREAD

**Discussion by luunak** [Cyfrin/foundry-full-course-cu#1951] [General]
hi, I am under impression of the courses quatlity, only question I have is if they are up to date, and secondly - updated? Seems like most of the information from creators are from like a year before.
So -  maybe the question to all of you who tried the course - do you think it it up to date regarding current tools, versions etc?

**Comment by urosognjenovic** [comment]
The courses have been updated to the latest best practices and tools a few days ago.

**Reply by luunak** [reply]
thanks, then they're even better :)

**Comment by EngrPips** [comment]
> hi, I am under impression of the courses quatlity, only question I have is if they are up to date, and secondly - updated? Seems like most of the information from creators are from like a year before.
> So -  maybe the question to all of you who tried the course - do you think it it up to date regarding current tools, versions etc?

Yeah it was updated few days ago and now seem like the best time to take the course.
